cancermuts/datasources.py

```diff
@@ -232,8 +232,10 @@ def _get_aliases(self, gene_id, to, fr='UniProtKB_AC-ID'):
             if len(responses) == 0:
                 self.log.warning(f"No {t} found for {gene_id}")
                 return None
+
             if len(responses) > 1:
-                raise TypeError
+            	self.log.warning(f"Multiple {t} found for {gene_id}. Entrez ID processing and mutation retrieval from cBioPortal will be skipped. Found {t}: {[response['to'] for response in responses]}")
+            	return None
```

Comment:
I don't think just returning `None` here works; normally we return a dictionary of responses, one per `t` in the loop. I would just set that one to `None`, and let the others as they are since they might just be valid, and then return the `out` dictionary as we already do